src/string.ts

```diff
@@ -9,17 +10,33 @@ export interface StringOptions {
 // the URLs of the pages where the string is used, forcing a newer version of a
 // string to use for multi-level translation selection, etc.
 export class TranslationString {
+  locale?: Locale;
   pod: Pod;
-  value: string;
   prefer?: string;
+  value: string;
 
-  constructor(pod: Pod, options: StringOptions) {
+  constructor(pod: Pod, options: StringOptions, locale?: Locale) {
     this.pod = pod;
-    this.value = options.value;
     this.prefer = options.prefer;
+    this.value = options.value;
+    this.locale = locale;
+  }
+
+  toLocale(locale: Locale) {
```

Comment:
Thoughts on calling this `localize`? It doesn't return a `Locale` object so the `toLocale` sort of doesn't make as much sense. FWIW, in Grow we have a `localize` method on `Document`s that return the localized version of the same document.

Reply:
+1 to this suggestion.

Reply:
I agree, that is a much better name. :)